scss/_buttons.scss

```diff
@@ -150,27 +150,19 @@
 
 // Boosted mod: border-only on :hover, :focus and :active
 .btn-no-outline {
```

Comment:
Hard to know if this is a good way to process or if a button-variant could the trick.